.github/workflows/macos-installer.yml

```diff
@@ -19,8 +19,6 @@ jobs:
         os:
           - macOS-latest
           - macOS-11
-        go:
-          - 1.20
```

Comment:
why we want to remove explict version because this make sure we have same on the CI which we use downstream?

Reply:
still testing things, for some reason the previous build failed, will remove this, plan is to keep pinning to 1.20

Reply:
removed now, the version needs to be in quotes like: '1.20' otherwise it thinks its version `1.2`, they have a note about it.. from their readme:

>Note: Due to the peculiarities of YAML parsing, it is recommended to wrap the version in single quotation marks:
```
  go-version: '1.20'
```
>The recommendation is based on the YAML parser's behavior, which interprets non-wrapped values as numbers and, in the case of version 1.20, trims it down to 1.2, which may not be very obvious. Matching an unstable pre-release: